Vizzuality/heco-invest
# LET-1268 Added updated_at to project and open call serializers

Needed on the FE.

## Testing instructions

since all project / open call endpoints use the same serializers, it should be enough to check e.g. the list endpoints

## Tracking

https://vizzuality.atlassian.net/browse/LET-1268
